gulpfile.js

```diff
@@ -10,46 +12,51 @@ const mergeStream = require('merge-stream');
 const polymerBuild = require('polymer-build');
 const browserSync = require('browser-sync').create();
 const history = require('connect-history-api-fallback');
+const eslint = require('gulp-eslint');
+const friendlyFormatter = require('eslint-friendly-formatter');
 
 const HtmlSplitter = polymerBuild.HtmlSplitter;
 const PolymerProject = polymerBuild.PolymerProject;
 const uglify = composer(uglifyes, console);
 
-const logging = require('plylog');
+// const logging = require('plylog');
```

Comment:
maybe just remove this, if we don't use it?

Reply:
we can use it if we want to get logs, so I would rather keep it for now